File: blog/20150508/spring_cache_jmx/src/main/java/sample/controller/SampleController.java
package sample.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.PathVariable;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.ResponseBody;
import sample.service.SampleService;

import javax.inject.Inject;

@Controller
public class SampleController {
	@Inject
	private SampleService sampleService;

	@RequestMapping("/sample/{id}")
	@ResponseBody
	public String sample(@PathVariable("id") String id) {
		return sampleService.sample(id);
	}
}

File: kotlin/spring/cloud/consul_sample/regist-sample/src/main/kotlin/sample/controller/SampleController.kt
package sample.controller

import org.springframework.cloud.client.discovery.EnableDiscoveryClient
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@EnableDiscoveryClient
class SampleController {
    @RequestMapping("/")
    fun sample() = "sample"
}